pyanaconda/ui/tui/spokes/network.py

```diff
@@ -51,16 +54,29 @@ def __init__(self, app, data, storage, payload, instclass):
         self.errors = []
 
     def initialize(self):
-        for name in nm.nm_devices():
+        self._load_new_devices()
+
+        EditTUISpoke.initialize(self)
+
+    def _load_new_devices(self):
+        """ Register new devices to supported devices list. """
+        devices = nm.nm_devices()
+        intf_dumped = network.dumpMissingDefaultIfcfgs()
+        if intf_dumped:
+            log.debug("Dumped interfaces: {0}".format(intf_dumped))
```

Comment:
I think we can drop using .format() for simple things like this. I used to encourage it because at one time it looked like py3 was going to deprecate % but in the end it really is cleaner for just printing something.